docs/api/DesktopAgent.md

```diff
@@ -138,6 +140,66 @@ agent.raiseIntent("StartChat", newContext, intentR.source);
 #### See also
 * [`IntentResolution`](#intentresolution)
 
+
+
+
+### `getOrCreateChannel`
+```typescript
+getOrCreateChannel(channelId: string): Promise<Channel>;
+```
+Returns a Channel object for the specified channel, creating it (as an _App_ channel) - if it does not exist.
+ `Error` with a string from the [`ChannelError`](Errors#channelerror) enumeration if channel could not be created or access was denied.
+
+#### Examples
+```javascript
+
+try {
+  let myChannel = await fdc3.getOrCreateChannel("myChannel");
+  myChannel.addContextListener(listener);
```

Comment:
This is not correct. Should be: `const listener = myChannel.addContextListener(context => { })`